funds-manager/funds-manager-server/src/custody_client/hot_wallets.rs

```diff
@@ -0,0 +1,42 @@
+//! Handlers for managing hot wallets
+//!
+//! We store funds in hot wallets to prevent excessive in/out-flow from
+//! Fireblocks
+
+use ethers::{
+    signers::{LocalWallet, Signer},
+    utils::hex::ToHexExt,
+};
+use rand::thread_rng;
+use tracing::info;
+
+use super::CustodyClient;
+use crate::{error::FundsManagerError, helpers::create_secrets_manager_entry_with_description};
+
+impl CustodyClient {
+    /// Create a new hot wallet
+    ///
+    /// Returns the Arbitrum address of the hot wallet
+    pub async fn create_hot_wallet(&self, vault: String) -> Result<String, FundsManagerError> {
+        // Generate a new Ethereum keypair
+        let wallet = LocalWallet::new(&mut thread_rng());
+        let address = wallet.address().encode_hex();
+        let private_key = wallet.signer().to_bytes();
+
+        // Store the private key in Secrets Manager
+        let secret_name = format!("hot-wallet-{}", address);
+        let secret_value = hex::encode(private_key);
+        create_secrets_manager_entry_with_description(
+            &secret_name,
+            &secret_value,
+            &self.aws_config,
+            "Hot wallet private key",
```

Comment:
nit: can we have this description (and perhaps the secret name) include which FB vault this hot wallet is for? is very helpful for discoverability if we ever need to manually find the secret